README.md

```diff
@@ -77,7 +83,24 @@ If you have the Doppler CLI installed, you can generate a Doppler Service Token
 kubectl create secret generic doppler-token-secret -n doppler-operator-system --from-literal=serviceToken=$(doppler configs tokens create doppler-kubernetes-operator --plain)
 ```
 
-Next, we'll create a `DopplerSecret` that references your Doppler token secret and defines the location of the managed secret.
+#### Option 2: OIDC Authentication
+
+First, ensure your cluster's OIDC discovery URLs are publicly accessible. Then create a Doppler [Service Account Identity](https://docs.doppler.com/docs/service-account-identities) with:
+
+- **Audiences**:
+  - `https://api.doppler.com`: Doppler host
+  - `dopplerTokenSecret:<namespace>:<secret-name>` where namespace and secret-name match your tokenSecret location (e.g., `dopplerTokenSecret:doppler-operator-system:doppler-token-secret`)
+- **Subject**: `system:serviceaccount:doppler-operator-system:doppler-operator-controller-manager` (the operator's shared ServiceAccount)
+
+Create a secret containing your OIDC configuration:
+
+```bash
+kubectl create secret generic doppler-token-secret -n doppler-operator-system --from-literal=identity=YOUR_IDENTITY_ID
+```
+
+### Creating the DopplerSecret
+
+Next, create a `DopplerSecret` that references your authentication secret and defines the location of the managed secret:
```

Comment:
I'm also realizing that this isn't going to work for OIDC flows, the `spec` will need the `project` and `config` fields set. They were omitted here previously because `doppler configs tokens create doppler-kubernetes-operator --plain` creates a config-scoped API token so the config being accessed is implicit. 

We should probably change the docs so that this command includes the `--project` and `--config` flag, and then the values used there are referenced in the DopplerSecret (here and in the sample spec files). I would use `example-project` and `prd`, as those would map to what a new user trying out Doppler would have created by default.